dandelion.json

```diff
@@ -635,6 +635,10 @@
       "path": "bundles/no-payment-credit-memo/",
       "version": "1.0.0"
     },
+    "notion-proxy-rest-api": {
+      "path": "bundles/notion-proxy-rest-api",
+      "version": "1.0.0.beta.1"
```

Comment:
https://semver.org/
```suggestion
      "version": "1.0.0-beta.1"
```

Reply:
@thx copy paste error from:
 "stock-api": {
      "path": "bundles/stock-api",
      "version": "1.0.0.beta.0"
    },

it is resolved